CancerGenomics.fsh

```diff
@@ -31,14 +31,9 @@ Description:    "Records an alteration in the most common DNA nucleotide sequenc
     GenomicDNAChange 0..1 MS and
     GenomicSourceClass 0..1 MS and
     AminoAcidChange 0..1 MS and
-    AminoAcidChangeType 0..1 MS and
+    AAChangeType 0..1 MS and
```

Comment:
Let's not change this. AminoAcidChangeType makes it easier to align with the component name as specified in the CG Reporting IG.  It also let's the implementer know that it is associated with the previous component AminoAcidChange.

Reply:
That's not the reason for this change. This is a workaround for a SUSHI bug https://github.com/FHIR/sushi/issues/122. We will change the name back (as noted in comments) when the bug is fixed.

Reply:
when will the bug be fixed?